src/eko/couplings.py

```diff
@@ -1,6 +1,10 @@
-r"""Contains the QCD beta function coefficients and the handling of the running coupling :math:`\alpha_s`.
+r"""Manage running (and non) couplings.
```

Comment:
```suggestion
r"""Manage running (and fixed) couplings.
```
do you mean like this? I'm confused about the language

Reply:
Yes: it was running and non-running (i.e. fixed). Suggestion accepted